src/mapping/tests/PetRadialBasisFctMappingTest.cpp

```diff
@@ -997,15 +1006,300 @@ BOOST_AUTO_TEST_CASE(testDistributedConservative2DV5Vector)
+                                     {-1, 0, {0, 1}, {2}},
+                                     {-1, 1, {1, 0}, {3}},
+                                     {-1, 1, {1, 1}, {4}},
+                                     {-1, 2, {2, 0}, {5}},
+                                     {-1, 2, {2, 1}, {6}},
+                                     {-1, 3, {4, 0}, {7}},
+                                     {-1, 3, {4, 1}, {8}}},
+                                    {{{0, 1}, 0}, {{2, 3}, 1}, {{4, 5}, 2}, {{6, 7}, 3}}),
+                  MeshSpecification({// The outMesh is local, distributed amoung all ranks
+                                     {0, -1, {0, 0}, {0}},
+                                     {0, -1, {0, 1}, {0}},
+                                     {1, -1, {1, 0}, {0}},
+                                     {1, -1, {1, 1}, {0}},
+                                     {2, -1, {2, 0}, {0}},
+                                     {2, -1, {2, 1}, {0}},
+                                     {3, -1, {4.1, 0}, {0}},
+                                     {3, -1, {4.2, 1}, {0}}},
+                                    {{{0, 1}, -1}, {{0, 1}, -1}, {{0, 1}, -1}, {{0, 1}, -1}}));
+}
+/*
+BOOST_AUTO_TEST_CASE(DistributedScaledConsistent2DV1Vector)
+{
+  PRECICE_TEST(""_on(4_ranks).setupMasterSlaves(), Require::PETSc);
+  Gaussian                        fct(5.0);
+  PetRadialBasisFctMapping<Gaussian> mapping(Mapping::SCALEDCONSISTENT, 2, fct, false, false, false);
+
+  testDistributed(context, mapping,
+                  MeshSpecification({// Consistent mapping: The inMesh is communicated
+                                     {-1, 0, {0, 0}, {1, 4}},
+                                     {-1, 0, {0, 1}, {2, 5}},
+                                     {-1, 1, {1, 0}, {3, 6}},
+                                     {-1, 1, {1, 1}, {4, 7}},
+                                     {-1, 2, {2, 0}, {5, 8}},
+                                     {-1, 2, {2, 1}, {6, 9}},
+                                     {-1, 3, {3, 0}, {7, 10}},
+                                     {-1, 3, {3, 1}, {8, 11}}},
+                                    {{{0, 1}, 0}, {{2, 3}, 1}, {{4, 5}, 2}, {{6, 7}, 3}}),
+                  MeshSpecification({// The outMesh is local, distributed amoung all ranks
+                                     {0, -1, {0, 0}, {0, 0}},
+                                     {0, -1, {0, 1}, {0, 0}},
+                                     {1, -1, {1, 0}, {0, 0}},
+                                     {1, -1, {1, 1}, {0, 0}},
+                                     {2, -1, {2, 0}, {0, 0}},
+                                     {2, -1, {2, 1}, {0, 0}},
+                                     {3, -1, {4.1, 0}, {0, 0}},
+                                     {3, -1, {4.2, 1}, {0, 0}}},
+                                    {{{0, 1}, -1}, {{0, 1}, -1}, {{0, 1}, -1}, {{0, 1}, -1}})
+                                    );
+}*/
```

Comment:
Does this test still fail?

Reply:
Unfortunately, yes. I cannot figure out why they fail. Setup is the same as the Eigen RBF one.